sqlmesh/core/plan/definition.py

```diff
@@ -272,19 +272,32 @@ def restatements(self) -> t.Dict[str, Interval]:
         return self._restatements
 
     @property
-    def loaded_snapshot_intervals(self) -> t.List[LoadedSnapshotIntervals]:
+    def loaded_snapshot_intervals(
+        self,
+    ) -> t.Tuple[t.List[LoadedSnapshotIntervals], t.List[Snapshot]]:
         loaded_snapshots = []
+        unloaded_snapshots = []
         for snapshot in self.directly_modified:
-            if not snapshot.change_category:
-                continue
-            loaded_snapshots.append(LoadedSnapshotIntervals.from_snapshot(snapshot))
+            if snapshot.change_category:
+                loaded_snapshots.append(LoadedSnapshotIntervals.from_snapshot(snapshot))
+            else:
+                logger.debug(f"Got an unloaded snapshot. Snapshot: {snapshot.name}")
```

Comment:
What's `unloaded snapshot`?

Reply:
A snapshot with no intervals and in this case because it doesn't have a change category. It is to contrast a `LoadedSnapshot` that is created within this method.